layers/shared_classes_layer/python/data_service.py

```diff
@@ -69,3 +69,65 @@ def delete_marker(self, markerId):
         except Exception as e:
             logger.error(f"Error deleting marker with markerId {markerId}: {e}")
             return None
+
+    def update_marker(self, marker: LocationMarker) -> LocationMarker:
+        """
+        Update an existing marker in DynamoDB
+        :param marker: A LocationMarker with updated location details. Replaces marker with identical ID
```

Comment:
change `location details` to information. Not just location can be updated

Reply:
Corrected